rest-dashboard.go

```diff
@@ -47,13 +47,29 @@ type BoardProperties struct {
 	Version    int       `json:"version"`
 }
 
+// GetDashboardByUID loads a dashboard and its metadata from Grafana by dashboard uid.
+func (r *Client) GetDashboardByUID(uid string) (Board, BoardProperties, error) {
+	return r.GetDashboard("uid/" + uid)
+}
+
+// GetDashboardBySlug loads a dashboard and its metadata from Grafana by dashboard slug.
+// Deprecated since Grafana v5
+//
+// For dashboards from a filesystem set "file/" prefix for slug. By default dashboards from
+// a database assumed. Database dashboards may have "db/" prefix or may have not, it will
+// be appended automatically.
+func (r *Client) GetDashboardBySlug(slug string) (Board, BoardProperties, error) {
+	path, _ := setPrefix(slug)
+	return r.GetDashboard(path)
+}
+
 // GetDashboard loads a dashboard from Grafana instance along with metadata for a dashboard.
 // For dashboards from a filesystem set "file/" prefix for slug. By default dashboards from
 // a database assumed. Database dashboards may have "db/" prefix or may have not, it will
 // be appended automatically.
 //
 // Reflects GET /api/dashboards/db/:slug API call.
-func (r *Client) GetDashboard(slug string) (Board, BoardProperties, error) {
+func (r *Client) GetDashboard(path string) (Board, BoardProperties, error) {
```

Comment:
We should probably hide this from now and users should migrate to `GetDashboardBySlug` since it does the same. Making the API sane and breaking it is **still** OK until we release 1.0.